third_party/move/move-prover/tests/sources/functional/bug_14762.exp

```diff
@@ -0,0 +1,9 @@
+Move prover returns: exiting with model building errors
+error: invalid transfer of references
+   ┌─ tests/sources/functional/bug_14762.move:45:26
+   │
+39 │         let (found, index) = find(&s.entries, |obj| {
+   │                                   ---------- It is still being borrowed by this reference
```

Comment:
This error seems overly conservative.  Shouldn't this reference `&s.entries` not outlive the call to `find` ?

Reply:
It is generated by V1 compiler. Do we want to take care of this?